### ISSUE TITLE
Autosomal recessive with genetic anticipation

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7487]
**Preferred term label:**
Autosomal recessive with genetic anticipation

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of autosomal recessive inheritance involving a gene that exhibits anticipation, the increase in severity and/or an earlier age of onset in subsequent generations.

**Parent term (use hpo.jax.org/app)**
Autosomal recessive inheritance HP:0000007

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts  Our decision was to move the 'anticipation' concept to the modifiers -- thus this issue becomes moot. I will move [Genetic anticipation HP:0003743](https://hpo.jax.org/app/browse/term/HP:0003743) to the new Inheritance modifiers section
